lib/nerve/reporter/zookeeper.rb

```diff
@@ -74,6 +73,8 @@ def stop()
     end
 
     def report_up()
+      log.info "nerve: adding zk node at #{@full_key}" if @full_key
```

Comment:
also, what is `@full_key` going to stringify into? i think reporting `@key_prefix` would be cleaner for human consumption unless you're specifically looking for some additional zk-related data.

Reply:
`/smartstack/global/server_side_rendering.gondola-themed-ads-management-portal/10-112-59-86-uswest1astagef_0000000050` is an example of `@full_key`. I need to check an example of `key_prefix` and see how it looks like

Reply:
yeah, that looks like useful info, let's use `@full_key`